src/unit_tests/remoted/test_secure.c

```diff
@@ -2225,20 +2235,7 @@ void test_router_message_forward_invalid_delta_json_message(void** state)
 
     router_syscollector_handle = (ROUTER_PROVIDER_HANDLE)(1);
 
-    expect_string(__wrap_router_provider_send_fb, msg, expected_message);
```

Comment:
Reason to discard messages ?

Reply:
These changes belongs to https://github.com/wazuh/wazuh/pull/28360

Reply:
ok, let's remove them, this pr is from string utils, so we can merge it, the string ones seem fine to me.